pkgs/applications/audio/ardour/default.nix

```diff
@@ -56,13 +56,13 @@
 }:
 stdenv.mkDerivation rec {
   pname = "ardour";
-  version = "6.9";
+  version = "7.0";
 
   # don't fetch releases from the GitHub mirror, they are broken
   src = fetchgit {
     url = "git://git.ardour.org/ardour/ardour.git";
     rev = version;
-    sha256 = "0vlcbd70y0an881zv87kc3akmaiz4w7whsy3yaiiqqjww35jg1mm";
+    sha256 = "CCwkXRNiVnrlxUn3QEt2e/w9aGdzZJ306O9/1ERj7/c=";
```

Comment:
Using a bare base64 hash is somewhat unusual, considering you're not getting it from upstream or something.

There's two ways we specify hashes in Nixpkgs:
1. `sha256 = "<base32>"`, which you've seen in the v6 derivation (you can convert to base32 with `nix-hash`)
2. `hash = "sha256-<base 64>"`, which is more generic.

There's really not a consensus as to what should be used yet (see https://github.com/NixOS/rfcs/pull/131), so really you can just pick which one to use here. I do know there are some maintainers encouraging migration even now, so maybe that'll help you decide? *shrugs*

Or maybe you just want to keep in-line with the v6 derivation, which is completely fine too.

Reply:
I've gone for the more generic version. Thank you for explaining it so clearly and giving some background on where the ecosystem is moving :)